### PR TITLE
Exported timeUnits in parts/Utilities.ts

### PR DESCRIPTION
**Pull request by marvin19**

### CODE REVIEW

File: ts/parts/Utilities.ts [hunk 1]
```diff
@@ -2378,7 +2378,7 @@ function animObject(
  *
  * @ignore
  */
-H.timeUnits = {
+const timeUnits = {
```
Comment (bre1470) [inline ts/parts/Utilities.ts]: If you do here a `const timeUnits: Highcharts.Dictionary<number> = {`, you can everywhere else get rid of the `timeUnits as any`.

Comment (bre1470) [reply ts/parts/Utilities.ts]: Also this https://github.com/highcharts/highcharts/blob/a8a330ff5800f81544c26228d095577c6f45d918/ts/parts/Utilities.ts#L144 has to be removed, as there is no `timeUnits` on `H` (Highcharts) set anymore.

File: ts/parts-gantt/GridAxis.ts [hunk 2]
```diff
@@ -681,7 +682,7 @@ addEvent(
                                 count = parentInfo.count * 10;
                             }
 
-                            unitRange = H.timeUnits[unitName];
+                            unitRange = (timeUnits as any)[unitName];
```
Comment (bre1470) [inline ts/parts-gantt/GridAxis.ts]: This `any` can be removed with proper typing.

File: ts/parts/DateTimeAxis.ts [hunk 3]
```diff
@@ -137,15 +139,15 @@ Axis.prototype.normalizeTimeTickInterval = function (
             null
         ]] as Array<[string, (Array<number>|null)]>,
         unit = units[units.length - 1], // default unit is years
-        interval = timeUnits[unit[0]],
+        interval = (timeUnits as any)[unit[0]],
```
Comment (bre1470) [inline ts/parts/DateTimeAxis.ts]: This `any` can be removed with proper typing.

File: ts/parts/DateTimeAxis.ts [hunk 4]
```diff
@@ -137,15 +139,15 @@ Axis.prototype.normalizeTimeTickInterval = function (
...
         multiples = unit[1],
         count,
         i;
 
     // loop through the units to find the one that best fits the tickInterval
     for (i = 0; i < units.length; i++) {
         unit = units[i];
-        interval = timeUnits[unit[0]];
+        interval = (timeUnits as any)[unit[0]];
```
Comment (bre1470) [inline ts/parts/DateTimeAxis.ts]: This `any` can be removed with proper typing.

File: ts/parts/DateTimeAxis.ts [hunk 5]
```diff
@@ -155,7 +157,7 @@ Axis.prototype.normalizeTimeTickInterval = function (
             var lessThan = (
                 interval *
                 (multiples as any)[(multiples as any).length - 1] +
-                timeUnits[units[i + 1][0]]
+                (timeUnits as any)[units[i + 1][0]]
```
Comment (bre1470) [inline ts/parts/DateTimeAxis.ts]: This `any` can be removed with proper typing.

File: ts/parts/Tooltip.ts [hunk 6]
```diff
@@ -1645,7 +1645,7 @@ H.Tooltip.prototype = {
             }
 
             // The first format that is too great for the range
-            if (timeUnits[n] > range) {
+            if ((timeUnits as any)[n] > range) {
```
Comment (bre1470) [inline ts/parts/Tooltip.ts]: This `any` can be removed with proper typing.

**Comment by TorsteinHonsi** [comment]
Pending CI testing

**Pull request merged**